cmd/openshift-install/gather.go

```diff
@@ -47,6 +47,7 @@ var (
 	gatherBootstrapOpts struct {
 		bootstrap string
 		masters   []string
+		sshKey    string
```

Comment:
Are we worried about the "user has separate keys authorized for the bootstrap machine and the masters" use case?  If so, I think we want a slice here, and a `StringArrayVar` below.

Reply:
I'm not sure how real that use-case is, but it simple enough to extend that I don't care that much to restrict... done with https://github.com/openshift/installer/compare/16c8fee3bd84fb272d50a107abc544c624bc26d6...4326be6c46e8e43025b370c7b7de9ef41872868b

Reply:
I was pointed at a QE cluster configured with different master keys once ;).